modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,483 @@
+= Configure a Customer-Managed VPC on GCP
+:description: Connect Redpanda Cloud to your existing VPC resources for additional security.
+:page-cloud: true
+
+You can connect Redpanda Cloud to your customer-managed virtual private cloud (VPC), and you can then create clusters with the most security. With standard BYOC clusters using a Redpanda-managed VPC, Redpanda creates a VPC in your Google Cloud account and manages its resources. With a customer-managed VPC:
+
+* You provide your own VPC in your Google Cloud account.
+* You maintain more control of your Google Cloud account, because Redpanda requires less permissions than standard BYOC clusters.
+* You control your security resources and policies, including subnets, service accounts, IAM roles, firewall rules, and storage buckets.
+
+[NOTE]
+====
+include::shared:partial$feature-flag.adoc[]
+====
+
+== Prerequisites
+
+* Redpanda creates a private Google Kubernetes Engine (GKE) cluster in the VPC. The subnet and secondary IP ranges you provide must allow public internet access. The configuration requires you to provide reserved CIDR ranges for the subnet and GKE Pods, Services, and master IP addresses. See <<Configure your VPC>>.
+* Redpanda requires access to certain Google APIs, storage buckets, and service accounts. See <<Configure the service project>>.
+* A standalone GCP project is recommended. If your host project (where your VPC project is created) and your service project (where your Redpanda cluster is created) are in different projects, you must first provision a shared VPC in Google Cloud. See the https://cloud.google.com/vpc/docs/provisioning-shared-vpc[Google documentation^]. 
+
+== Limitations
+
+* Existing clusters cannot be moved to a customer-managed VPC.
+* After creating a cluster with a customer-managed VPC, you cannot change the VPC.
+
+== Configure your VPC
+
+. Create the primary and secondary subnets in your VPC using CIDR notation. Redpanda clusters require one subnet, and that subnet should have two secondary IP ranges:
++
+--
+* Subnet IP range should be at least /20 CIDR, such as 10.0.0.0/20.
+* Secondary IP for GKE Pods is a /21 CIDR, such as 10.0.8.0/21.
+* Secondary IP for GKE Services is a /24 CIDR, such as 10.0.1.0/24.
+--
++
+Replace all `<placeholders>` with your own values.
++
+```bash
+gcloud compute networks subnets create <primary-subnet-name> \
+    --project <host-project-id> \
+    --network <shared-vpc-name> \
+    --range 10.0.0.0/24 \
+    --region <region> \
+    --secondary-range <secondary-ipv4-range-name-for-pods>=10.0.8.0/21,<secondary-ipv4-range-name-for-pods>=10.0.1.0/24
```

Comment:
The second variable on this line should be `secondary-ipv2-range-name-for-services`